invenio_app_ils/records/loaders/schemas/json/series.py

```diff
@@ -7,15 +7,25 @@
 
 """Series schema for marshmallow loader."""
 
+from invenio_records_rest.schemas import RecordMetadataSchemaJSONV1, \
+    StrictKeysMixin
 from invenio_records_rest.schemas.fields import PersistentIdentifier
 from marshmallow import Schema, fields
 
 
-class SeriesSchemaV1(Schema):
+class SeriesSchemaV1(RecordMetadataSchemaJSONV1, StrictKeysMixin):
     """Series schema."""
 
+    def get_pid_field(self):
+        """Return pid_field value."""
+        return "pid"
+
+    title = fields.Str(required=True)
     pid = PersistentIdentifier()
     mode_of_issuance = fields.Str()
```

Comment:
`mode_of_issuance` is required too. @kprzerwa maybe you can help with the mandatory fields.
I think that the loader should then match the required fields in json schema.

Reply:
I am not sure this has been fixed